src/mame/mr/dribling.cpp

```diff
@@ -34,32 +32,20 @@
 
 ***************************************************************************/
 
-#include "emu.h"
 
+#include "emu.h"
+#include "machine/netlist.h"
+#include "netlist/devices/net_lib.h"
 #include "cpu/z80/z80.h"
 #include "machine/i8255.h"
+#include "screen.h"
+#include "speaker.h"
 #include "machine/watchdog.h"
-
 #include "emupal.h"
 #include "screen.h"
-
-
-// configurable logging
-#define LOG_MISC     (1U << 1)
-#define LOG_SOUND    (1U << 2)
-#define LOG_PB       (1U << 3)
-
-//#define VERBOSE (LOG_GENERAL | LOG_MISC | LOG_SOUND | LOG_PB)
-
-#include "logmacro.h"
-
-#define LOGMISC(...)     LOGMASKED(LOG_MISC,     __VA_ARGS__)
-#define LOGSOUND(...)    LOGMASKED(LOG_SOUND,    __VA_ARGS__)
-#define LOGPB(...)       LOGMASKED(LOG_PB,       __VA_ARGS__)
-
+#include "nl_dribling.h"
 
 namespace {
-
 class dribling_state : public driver_device
```

Comment:
You’re including two headers twice, you’ve lost the blank lines between header groups, you’ve lost the double blank line between separating the headers and logging helpers from the code, and you’ve lost the blank line between the start of the anonymous namespace and the class declaration.

Reply:
I see "screen.h" being included twice. But that's the only one. I am sorting them alphabetically to make sure (if sort order is irrelevant).

Reply:
Double checked, no duplicates.